builders/get-started/endpoints.md

```diff
@@ -27,13 +27,14 @@ description: 使用支持的API提供商连接至公共端点或创建私有JSON
 
 --8<-- 'text/endpoints/moonbase.md'
 
-## 端点提供商 {: #endpoint-providers } 
+## RPC端点提供商 {: #endpoint-providers }
 
 您可以使用以下任意API提供商，创建适用于开发环境或生产环境的端点：
 
 - [1RPC](#1rpc)
 - [Blast](#blast)
 - [BlockSpaces](#blockspaces)
```

Comment:
At quick glance, Blockspaces doesn't seem to provide Web3 RPC services anymore. If that's the case we should remove them from the CN and english sites

Reply:
oh yeah good catch!! they deprecated it as of 6/1/2023. I'll remove it here and open a PR to remove it on the EN repo